src/basic/renderers/select-idp.ts

```diff
@@ -0,0 +1,183 @@
pyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgpMwidZAAAHuUlEQVRYCdWYy44kRxWGT2RmdbunbXxDBhvEAoHYWcKSJVvymifgIXgJduy9Q7wBW5aW7Z2XMBuEYIEMsuWL8G2Yme6uS2dm8H9/ZFRnlavGGFkNRHdmRpw45z//iYiMOFnp/m9f/3X/z49fjKuLHGMbKaXI+otowrWkeqNmLeMY7p7aY1Kl7aJ5uIzmJy/G4z//Rfz9/tvxh/d+E+cnd2ITo5FkdaTgC51Wz1JHt8HnkOM7T74U3epPd185f+/PPx3WV9JBNcXQQLMxtzHNSEEInKyb6ynapjfi+OnnsW4fi6eS0Idl/OX938WdJ74rrGV0eMXuESXJdxJZgs6qJfkd1+vY9FfRnQ2XF60I5s0yp9wlD1waTBIeDSR3CtKb0lgzR9/fi+XVMrJG+uLiMjaDBvj6YfRxGe1NTDZ0fKrhC+5coiViGpxJxowOfY7LzWfRqVuUxsipsXeUs6pE4yLllEu9gBXx9j4utEoEqKtvN8DF0PVqlylrR4XR7AYKDkWD52J8fOBG9rDGf45B9xaSoyOAqrUy0ysVwnThyQX9/QKU7NSdRi0QjzpLhbBBLqub/inOXQApFU6TD/RQthAOUtetiyyeCik7LEs1chjN3xY1EZUblakw2fIk1aRbGoXlWhetxKzFvq5f1Q8WcOW2eCYwefEglUHh3uFkFFuuRuE6ejm1rZTLExTVHNrMlUTYQZAXrZYRHJlojiTSBYjrrlQ1yyzxoOCjRIKVhx65/gld5qiioNGQBkSrC5MrXiZn6topRCSiWiZ17ZqXxFqOkbWNlFKfc2PJzEh6qlYNFhGcSjsXkqzrTqHTiXIZm2LtdrWe4091v5Gu131OoYp0eVeMeHg9Viy5AZ4AubTF0DAXM9BNi0cRS4jAU830ubdEYiP3VtTdJ6FRsqe9TJfXKTI6jEXlSIGT/6oqiOIjeUFmMemtvCkzREV1m6WyqM+575vVPpf+F+p1SOpzTuF/huSc1H79/4XkoQEmlkOrYz/Gb759yCs72RFPx8jvq2NfMapNfe7rHmvXPbG+4dLbQnif7Mr+JEc+r+1vMnJd2jUTOOCDkyYl5ZPjiRB8Ngi/t4+tH+zA2hFMMovZdpQFeCsUIkmJdL015oHjtRxog4TexAXG3piVdbJXtUY3W8F9uRT+vU4XHa/kluDpBGAzz2nhy6cdBPdJIrI/6YrV2Cible1CBwtZVN8KLy8U+uoyLi4/js1GOWWciRRRXKq+kDPO5LWwOYUPEdWYiY1zlAcRm5VSNWmudRRerRRgc9/bMOe4S31OTSDnIn0Y+PgeFOCghbgSx9X5JtLm7tvvpOv1a6vlUqba3XVyZIXuY8rUCrl9kkhx4NNFLDOIzz0fpy+9HJ9cvRsffnY3TuSojJRmZs5mIkkfpR6txpQieUBOrTKoIZ48eyHSvXF854mUXrvg5JkBzaoF6cidPQxdzFsF2GrkT83sGqmueqabgtr7BTkFPZBoY3eqiyT6LLpf/f5efPBgE8uV83NFwBrlxfEK9UiwoA8XgSrapOTxi/UYP3vusfjlqy/E+NGbcfm31yOdLqIdTko2ZIxKAjRsJ+Lqw6u/cyRifafMSI5x59kfRffWg0X88R8nEVeaLn2A1ZyuzA/jhIys/VDBqaKGwNUY5wtGQNf6r9F98kYszk71Aq3L4BwyrzL4TvyBch2ZoJr8SnTPKO391mmKZWgLaURGV2Klo6wpKOmT5KRGjnxCnkD5pmSilsMqzhZMscqJRnChFGvBtqRRgfujyoRlFfkVpIregl5cusf1dktIit9nXneh8WqpoEtUkCzkkHy5sIX5pdJXIlsP4OVzRNWR/VJfnpAwIs/d4i73iZldaOrxKzX2ySat+HyYQNO1O5L2gcz08vFon3zeFqhd+NoatHWDyo5aloVp24SbWnZe9Y88NdzOSTVG1RtPskkP7NbMpNyzFd2YzER71QL67zDZM/yq5gS5S7KG8FXGt9FfhtGedknehvOv4aOs5ek9+hp2t6pa1/LuSNZlVZ+3SmnPGRwOrsmq95+szTo3FeObeE48SgJYARHOIqjiR/snR1LRacWeWco8SskeBTBXtXElUZDYzpxPTru2pds9TguCP/5tVv0X2y0cCuyQEJ37q/XyMwx9RptZ3+iTk9bDrNoRFzluq76O05+D3dm3CbFMBcgx6GNR7oXA74WHC/Sw5151+MmuiGFfpEpYqNRg53Xw0YeZ2aqtw8QyGXT8wEnWQ0aMsCirUn80kKD+wgE+/fOnmtOJQGIMYUohjSLY2+nGsJZtXWRqtus+9FURVM13ukFpVi/PJBeFANNSjiid6goMC0a2oFdsiyeZDZPOfqNPekyIilOwqldEe/dZZ/3OQKSLPKfVn3/O5tw1dfWQY3hobEuUumA2GZoH9cJd5KnIKCvdM3PAr+1AHTKDLaAVBOm8SM4HkXBK3s2MMv4sHnKBMbpeicqmF8lrJb3VOZgmKWj8T/U59E1dUPpYCmVpeeAbh8xFDzB6jMlTVTcGRGelNid81qwHU06TiJt7PIzuh2fj6erbi7jY8Kpryh2wVpcNealwe7wQb9JH26fny/j+s3jRJ8Sd78Xp0z/QXJ3LWDIW/lEU9dunfOOTX4s9KhpRpWnx1I/jX6AtWM7LAAtKAAAAAElFTkSuQmCC';
```

Comment:
These are the fallback images as base64 encoded dataURIs for the 4 most common providers.